### PR TITLE
Release v0.0.8

### PR DESCRIPTION
**Pull request by alex-stone**
### What changed? Why?

* Enable ethereum and base network asset creation
* Cleans up error handling to surface server-side errors in a more parseable manner
* Adds dynamic support for assets for balances, transfers, and trades.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- Remove unused `base_sepolia_rpc_url` configuration option.
+- Support assets dynamically from the backend without SDK changes.
+
```
Comment (John-peterson-coinbase) [inline CHANGELOG.md:12]: Lets move these to a `0.0.8` block.

**Review by John-peterson-coinbase — COMMENTED** [review]
One comment on CHANGELOG.md. LGTM and can +1 after that!

**Review by jazz-cb — APPROVED** [review]

**Comment by cb-heimdall** [comment]
This Pull Request was merged without enough reviews
Please go [here](https://github.com/coinbase/coinbase-sdk-ruby/issues/94) to review and +1

**Comment by cb-heimdall** [comment]
This Pull Request was merged without enough reviews
Please go [here](https://github.com/coinbase/coinbase-sdk-ruby/issues/94) to review and +1

**Comment by cb-heimdall** [comment]
This Pull Request was merged without enough reviews
Please go [here](https://github.com/coinbase/coinbase-sdk-ruby/issues/94) to review and +1

**Pull request merged**